src/main/java/com/avispl/symphony/dal/avdevices/microphone/sennheiser/tcc2/dto/meter/input/InputDTO.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *  * Copyright (c) 2023 AVI-SPL, Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.avdevices.microphone.sennheiser.tcc2.dto.meter.input;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/**
+ * InputDTO class includes peak value
+ *
+ * @author Kevin / Symphony Dev Team<br>
+ * Created on 3/15/2023
+ * @since 1.0.0
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class InputDTO {
+	private int peak;
```

Comment:
Is it used to transfer the Json data, if not please remove the @JsonIgnoreProperties(ignoreUnknown = true).